src/pretix/plugins/paypal/payment.py

```diff
@@ -398,9 +410,13 @@ def _execute_payment(self, payment, request, payment_obj):
                     "value": {
                         "items": [
                             {
-                                "name": ('{} '.format(self.settings.prefix) if self.settings.prefix else '') +
-                                __('Order {slug}-{code}').format(
-                                    slug=self.event.slug.upper(), code=payment_obj.order.code
+                                "name": '{prefix}{orderstring}{postfix}'.format(
+                                    prefix=self.settings.prefix if self.settings.prefix else '',
+                                    orderstring=__('Order {slug}-{code}').format(
+                                        slug=self.event.slug.upper(),
+                                        code=payment_obj.order.code
+                                    ),
+                                    postfix=self.settings.postfix if self.settings.postfix else ''
```

Comment:
Hmm, that will look weird:

``PREFIXOrder ABC-1234POSTFIX``

The old code added a space after the prefix, I think we should keep that and add it before the postfix and in the other places as well.

Reply:
Sorry, apparently I did not read the full PR body, you explain it. I'm not sure what the best way forward is.

A migration would solve it but only caus problems in the future, IIRC our input fields usually strip whitespace so it would be impossible to configure a new prefix with a trailing space.

Is there really a reason that makes us *need* to drop the whitespace?

Reply:
Weird magical idea: Auto-add whitespace after the prefix / before the postfix if it ends/starts alphanumeric.

i.e. if your prefix is ABC, do "ABC EVENT-ORDER", but if your prefix is ABC-, do "ABC-EVENT-ORDER".

Would probably be the desired thing in all practical situations, but it would be a bit surprising behaviour

Reply:
Feeling lazy - omitting the magic for now...